### ISSUE TITLE
closeDrawer is missing from course.model.schema

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#1991]
drawer.hbs references `_globals._accessibility._ariaLabels.closeDrawer` but this is missing from course.model.schema

Additionally course.json contains a reference to `drawerBack` but this is not actually used (`_ariaLabels.previous` is assigned to the drawer back button)
